web_ui/frontend/components/layout/Sidebar/RandomUserIcon.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import React, {useMemo} from "react";
+import {Face2, Face3, Face4, Face5, Face6} from "@mui/icons-material";
+
+const RandomUserIcon = () => {
+
+    // Pick a random icon for the user
```

Comment:
I noticed the jumpiness of the user icon every time I refresh/hit a new page and I'm pretty confused. Can we fixed it to a generic user icon?